src/main/java/org/broadinstitute/hellbender/tools/walkers/CombineGVCFs.java

```diff
@@ -34,35 +34,33 @@
  * Combine per-sample gVCF files produced by HaplotypeCaller into a multi-sample gVCF file
  *
  * <p>
- * CombineGVCFs is meant to be used for hierarchical merging of gVCFs that will eventually be input into GenotypeGVCFs.
- * One would use this tool when needing to genotype too large a number of individual gVCFs; One would first use
+ * CombineGVCFs is meant to be used for merging of GVCFs that will eventually be input into GenotypeGVCFs.
+ * One could use this tool when needing to genotype multiple individual GVCFs instead of GenomicsDBImport; one would first use
```

Comment:
The main advantage of `CombineGVCFs` over `GenomicsDBImport` is the ability to combine multiple intervals at once from the GVCFs (`GenomicsDBImport` can only combine one interval per invocation). The main disadvantage is slower speed, which may not be an issue when there are only a few samples.